backend/__tests__/userDeleteBooking.test.js

```
@@ -0,0 +1,331 @@
+  // Return a modified version that includes our own deleteBooking implementation
+  return {
+    ...actualModule,
+    deleteBooking: jest.fn()
+  };
+});
+
+// Import the mocked version
+const { deleteBooking } = require('../controllers/userControllers');
+
+describe('deleteBooking Controller', () => {
+  let req, res, userId, bookingId, propertyId, propertyOwnerId;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)